app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -76,7 +82,7 @@ const AppLayoutWithExperiment = ({
   if (
     !loading &&
     redirect &&
-    (getRedirect = redirect!({ status, review, analysis }))
+    (getRedirect = redirect!({ status, review, analysis, analysisError }))
```

Comment:
I don't think `analysisError` is being used in any of the `redirect` prop calls

Reply:
Oh yep, good catch